FuncGodot Manual/pages/tips_runtime_building.html

```diff
@@ -0,0 +1,126 @@
+        <blockquote><i>NOTE: Godot 4.2 does not currently support runtime lightmap baking or UV2 unwrapping in exported builds. For now, consider using a different lighting solution for runtime building.<br></i></blockquote>
+        An example of runtime map building is as follows:</p>
+        <pre><code>extends Node
+
+@onready var func_godot_map := $FuncGodotMap as FuncGodotMap
+
+# We can connect the various build signals of the FuncGodotMap 
+# in order to add more functionality after building.
+func _ready():
+    func_godot_map.connect("build_complete", _build_complete)
+    func_godot_map.connect("build_failed", _build_failed)
+    func_godot_map.connect("unwrap_uv2_complete", _unwrap_uv2_complete)
+    func_godot_map.local_map_file = "res://maps/runtime_building.map"
+    func_godot_map.verify_and_build()
+
+# We need to wait for the build to finish before unwrapping mesh UV2s for lightmap baking.
+# Currently lightmap baking cannot be performed at runtime, so it's not terribly useful yet.
+# Consider creating a procedural Voxel GI solid class entity that can be baked at runtime instead.
+func _build_complete() -> void:
+    print("Success! Unwrapping UV2...")
+    func_godot_map.unwrap_uv2()
+
+func _build_failed() -> void:
+    printerr("Failed to build the map file! :(")
+
+func _unwrap_uv2_complete() -> void:
+    print("UV2 unwrapping completed!")</code></pre>
+    <p>An example of this can be found in <a href="https://github.com/func-godot/func_godot_runtime_building" target="_blank">func_godot_runtime_building repository</a>.</p>
+
+    <h1>Building from Map Editors</h1>
+    <p>
+        Much like idTech engine games, it's possible to build and launch directly into a map from some map editors.
+        This can be quite a helpful tool for testing rapid iterations.
+    </p>
+
+    <h2>Godot Project Setup Example</h2>
+    <p>To build and launch our maps from the editor, you'll need to employ the <a href="https://docs.godotengine.org/en/stable/classes/class_os.html#class-os-method-get-cmdline-user-args" target="_blank">OS class' ability to retrieve command line arguments</a>. A basic example might look something like this:</p>
+
+
+    <code><pre>
+# This script should be autoloaded to handle command line arguments
+# Additionally, requires a valid reference to a FuncGodot map
+
+var func_godot_map: FuncGodotMap
+
+var map_override: String
+var cmd_line_args := OS.get_cmdline_user_args()
+
+# Loop over all command line arguments, split on a delimiter for each command, and use this for the file path
+for cmd_line_arg in cmd_line_args:
+    if cmd_line_arg.contains("launch_debug::"):
```

Comment:
Can we come up with a command name that's more obvious than `launch_debug`? Something like `launch_map` or `run_map` or something? Something that says "this command is meant to start the game on the map we have open" when adding it to the command line parameters in the editors?

Reply:
`buildmap <path>` maybe? I can see why this would be confusing looking back?

I think any of the above would be fine, really.

Reply:
I thought of `buildmap` but it does more than just build the map. Kind of why I settled on `launchmap` or `runmap` since that's the ultimate goal and building the map is more of a requirement towards that goal.

Reply:
Personally, I don't have a strong opinion on this. If we change this, we have to take new screenshots for all the editor variants.

Reply:
If we keep the focus on editor project launching rather than exported exe launching, we'll have to take new screenshots for all the editor variants anyway.

But I have a strong opinion on it only for the sake of keeping things as clear as possible for a semi-complicated subject. It is a tutorial after all.

My vote is for `run_map`. With the underscore. It's concise, it does what it says on the tin (and all the tasks needed to do what it says). Also the "n" to "m" flows much nicer than "ch" to "m" imo.